docker/wrapper.py

```diff
@@ -56,8 +56,25 @@ def call_pipeline(mount, args):
         else:
             log.info('Flag "--no-clean" was used, therefore {} was not deleted.'.format(work_dir))
 
+    #DEBUG !!!
+    print("DEBUG!!!!! output dir is "+mount+" and files are:")
+    print(os.listdir(mount))
+    fail_files = [fail_file for fail_file in os.listdir(mount) if fail_file.startswith("FAIL.") and fail_file.endswith(".tar.gz")]
```

Comment:
`fail_files = glob.glob(os.path.join(mount, 'FAIL*'))`
or
`fail_files = [os.path.join(mount, x) for x in os.listdir(mount) if x.startswith('FAIL')]`

PyCharm's PEP is lines < 120 characters.